src/components/settings/EditOrganizationInformationsDialog.tsx

```diff
@@ -0,0 +1,323 @@
+          </Button>
+          <Button
+            variant="primary"
+            disabled={!formikProps.isValid}
+            onClick={async () => await formikProps.handleSubmit()}
+          >
+            {translate('text_62ab2d0396dd6b0361614db2')}
+          </Button>
+        </>
+      )}
+    >
+      <Content>
+        <FormSection>
+          <AvatarContainer>
+            {logo || organization?.logoUrl ? (
+              <Avatar size="large" variant="connector">
+                <img
+                  src={(logo || organization?.logoUrl) as string}
+                  alt={`${organization?.name}'s logo`}
+                />
+              </Avatar>
+            ) : (
+              <Avatar
+                size="large"
+                variant="company"
+                identifier={organization?.name || ''}
+                initials={(organization?.name || '')
+                  .split(' ')
+                  .reduce((acc, n) => (acc = acc + n[0]), '')}
+              />
+            )}
+            <AvatarUploadWrapper>
+              <Button variant="secondary" onClick={() => hiddenFileInputRef?.current?.click()}>
+                {translate('text_62ab2d0396dd6b0361614d18')}
+              </Button>
+              <Typography variant="caption" color={logoUploadError ? 'red' : ''}>
+                {translate('text_62ab2d0396dd6b0361614d20')}
+              </Typography>
+            </AvatarUploadWrapper>
+            <input
+              type="file"
+              accept="image/png, image/jpeg"
+              ref={hiddenFileInputRef}
+              onChange={(event: React.ChangeEvent<HTMLInputElement>) => {
+                setLogoUploadError(false)
+                const file = event?.target?.files?.[0]
+
+                if (file) {
+                  getBase64(file)
+                }
+              }}
+              style={{ display: 'none' }}
+            />
+          </AvatarContainer>
+        </FormSection>
+        <FormSection>
+          <TextInput
+            name="name"
+            label={translate('text_62ab2d0396dd6b0361614d5c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d5c')}
+            value={organization?.name || ''}
+            disabled={true}
+          />
+        </FormSection>
+        <FormSection>
+          <TextInputField
+            name="legalName"
+            label={translate('text_62ab2d0396dd6b0361614d6c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d6c')}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus
+            formikProps={formikProps}
+          />
+        </FormSection>
+        <FormSection>
+          <TextInputField
+            name="legalNumber"
+            label={translate('text_62ab2d0396dd6b0361614d7c')}
+            placeholder={translate('text_62ab2d0396dd6b0361614d7c')}
+            // eslint-disable-next-line jsx-a11y/no-autofocus
+            autoFocus
```

Comment:
You already have an "autoFocus" on `legalName`, does not make sense to have 2 of them

Reply:
yeah and it bring a wrong behaviour. Removed